src/ert/storage/local_experiment.py

```diff
@@ -335,6 +335,43 @@ def response_key_to_response_type(self) -> Dict[str, str]:
         mapping = {}
         for config in self.response_configuration.values():
             for key in config.keys:
+                if key == "*":
+                    continue
```

Comment:
Any way we can avoid this, and rather replace it once we know what the actual responses are called? My main concern is that we are propagating the wilcard to far into the logic.

Reply:
Hmm I think we could maybe get rid of it yes, should be some property of the respones config I'd say. If we get rid of it now, then the information that we accept wildcards is no longer there. Does it make sense that we only account of the "*" on the "read from runpath to storage step" and ignore it afterwards, and always read all keys that make it into storage?

Reply:
I think that makes sense, I guess that will already work with the changes you have here? As you would update the keys the first time we save something from runpath?